### ISSUE TITLE
hyper > HTTP20Connection > HTTPConnection > request_chunked

### ISSUE THREAD

**Issue by 1993-tensa-g** [lddias/python-avs#2]
Not an issue ... just question

hyper doesn't have this function available.

we can use normal request of of http20 but that will change whole handling process right?

**Comment by lddias** [comment]
Right, I made a fork of hyper which has this function and other changes to take advantage of the better streaming/low-latency of AVS over HTTP2. If you install the requirements via pip, you can see the fork being used in requirements.txt:

`-e git://github.com/lddias/hyper.git@development#egg=hyper`
